tox.ini

```diff
@@ -57,6 +57,7 @@ commands =
 [testenv:docs]
 deps =
   -c dev-requirements.txt
```

Comment:
Do we still need `dev-requirements.txt` in the `doc` env?

Reply:
Yes, at least pytest is needed

Reply:
Never mind looks like there is some duplication of packages in requirement files, will clean up a little bit